tests/bug-5220/bug-5220-2/README.md

```diff
@@ -0,0 +1,8 @@
+# Test Description
+This test demonstrates an invalid combination of base64_data with fast_pattern.
```

Comment:
Looks like the same as first test to me. What is the added value ?

Reply:
Not much with the way `base64_data` is setup but I tried to create different combinations that are allowed rn w.r.t. the `base64_data` and `fast_pattern` keywords to see if anything would happen differently.
Do you think this test should be removed?

Reply:
Maybe rather more in the README to explain the difference : multiple `base64_data` with one which is fast_pattern... or such

And why it matters : I would expect rule parser to stop after fast_pattern and not care about the end of the rule...

Reply:
You're right about the why part. I think I added all these different combinations for myself to understand things better and walk code in diff scenarios. I'll fix the README and also add that the rule parser stops after the fast_pattern. Thank you!